cmd/immudb/command/init.go

```diff
@@ -89,6 +89,7 @@ func (cl *Commandline) setupFlags(cmd *cobra.Command, options *server.Options) {
 	cmd.Flags().String("s3-path-prefix", "", "s3 path prefix (multiple immudb instances can share the same bucket if they have different prefixes)")
 	cmd.Flags().Bool("s3-external-identifier", false, "use the remote identifier if there is no local identifier")
 	cmd.Flags().String("s3-instance-metadata-url", "http://169.254.169.254", "s3 instance metadata url")
+	cmd.Flags().String("s3-fargate-credentials", "false", "s3 fargate credentials true/false")
```

Comment:
Can you change this to:

```golang
cmd.Flags().Bool("use-fargate-credentials", false, "automatically pull AWS credentials from Fargate task metadata")
```
?

I think it better matches the intent of the switch.